### ISSUE TITLE
Hyperparameter --lr_finetuning_init and --lr_unlearning_init seems to be not used in  ./defense/abl/abl.py

### ISSUE THREAD

**Issue by mo666666** [SCLBD/BackdoorBench#12]
Thank you for your excellent work. I found that the hyperparameters --lr_finetuning_init and --lr_unlearning_init seem to be not used in the **./defense/abl/abl.py**. Is that a bug?

**Comment by mdzhangst** [comment]
Yes, because the experiment found that unlearn had a high sensitivity to lr, so the lr given in the original text was retained without using hyperparameters. In order not to affect you, we will make changes in later versions.

**Comment by mo666666** [comment]
Ok, thank you for your replies!
